extensions-rhel-9.0.yaml

```diff
@@ -53,9 +53,6 @@ extensions:
   sandboxed-containers:
     architectures:
       - x86_64
-    modules:
-      enable:
-        - virt:rhel
```

Comment:
This probably means that every cluster will need a RHEL subscription configured by default to be able to fetch those packages.

Reply:
One thing that we discovered as part of layering work is there's actually *today* a magical service that turns the container pull secret into entitlement certificates.  It's just not on by default - it's intended mainly for container builds.  But now that there's a *reason* - perhaps we can just push to turn it on by default (for connected clusters).

Among other things, it would make `yum install foo` also Just Work inside toolbox and `oc debug node`.